netty/src/main/java/io/grpc/netty/ProtocolNegotiators.java

```diff
@@ -904,8 +904,7 @@ protected ProtocolNegotiationHandler(ChannelHandler next) {
 
     @Override
     public final void handlerAdded(ChannelHandlerContext ctx) throws Exception {
-      InternalProtocolNegotiators.negotiationLogger(ctx)
-          .log(ChannelLogLevel.DEBUG, negotiatorName + " started");
+      negotiationLogger(ctx).log(ChannelLogLevel.DEBUG, negotiatorName + " started");
```

Comment:
Avoid string concat. But no need do it in this PR.

Reply:
I'll do a followup.